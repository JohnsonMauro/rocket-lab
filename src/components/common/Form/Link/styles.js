import styled from '@emotion/styled';

const LinkStyles = styled.a`
  color: ${({ theme }) => theme.colors.blackRussian};
  font-size: 14px;
  line-height: 20px;
  opacity: 0.4;
`;

export { LinkStyles };
