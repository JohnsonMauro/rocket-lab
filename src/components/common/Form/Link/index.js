import { LinkStyles } from './styles';

const Link = ({ children }) => (
  <LinkStyles href="">{children}</LinkStyles>
);

export default Link;
