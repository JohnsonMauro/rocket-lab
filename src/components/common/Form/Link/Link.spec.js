import React from 'react';
import { render } from '@testing-library/react';

import GlobalsContainer from '@test/GlobalsContainer';
import Link from './index';

describe('<Link />', () => {
  it('Renders the Link component', () => {
    const { container } = render(<Link>Sample</Link>, {
      wrapper: GlobalsContainer,
    });
    expect(container).toMatchSnapshot();
  });
});
