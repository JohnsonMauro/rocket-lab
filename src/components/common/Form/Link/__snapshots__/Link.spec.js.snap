// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> Renders the Link component 1`] = `
<div>
  <a
    class="css-1fwtyfg-LinkStyles e12g151z0"
    href="#"
  >
    Sample
  </a>
</div>
`;
